### DISCUSSION TITLE
just submitted a new issue and was given missing template label

### DISCUSSION THREAD

**Discussion by mawi675lr81** [bromite/bromite#1664] [Q&A] [answered]
How do I correct it?

**Comment by ghost** [comment]
which issue you reported?, i can help in making it correct

**Reply by mawi675lr81** [reply]
https://github.com/bromite/bromite/issues/1663

**Reply by ghost** [reply]
> #1663

you should put the urls of websites u faced the bug on in “steps to reproduce the bug”

**Reply by mawi675lr81** [reply]
But it's not an issue with the URL. I'll try again though. Should I create a new issue or will I be able to update the existing one?

**Reply by mawi675lr81** [reply]
True to update as well as creating a new one. Didn't work. New one still shows missing template and duplicate. Reporting a bug should not be this difficult

**Comment by mawi675lr81** [comment]
@csagan5 @uazo please look into it. I don't know what else I can do to correct the template

**Reply by uazo** [reply]
I do not know, try to edit:
```
Version: 96.0.4664.54 Arch: arm64`
-----------------------------^^^
```
in any case it is difficult to reproduce your bug without having a device like yours

**Reply by mawi675lr81** [reply]
Please let me know if there's something i can do to help

**Reply by csagan5** [reply]
You have removed the question about privacy.

**Comment by mawi675lr81** [comment, marked as answer]
It looks like when 'close all open tabs on exit' is on, closing the fold is triggering the browser as exit. I just turned it off and folded the screen, and I can see the transition happen to the small screen and vice versa.
